ibis/backends/pandas/__init__.py

```diff
@@ -90,11 +96,58 @@ def load_data(self, table_name, obj, **kwargs):
         self.dictionary[table_name] = obj
 
     def get_schema(self, table_name, database=None):
-        return sch.infer(self.dictionary[table_name])
+        schemas = self.schemas
+        try:
+            schema = schemas[table_name]
+        except KeyError:
+            schemas[table_name] = schema = sch.infer(
+                self.dictionary[table_name]
+            )
+        return schema
 
     def compile(self, expr, *args, **kwargs):
         return expr
 
+    def create_table(self, table_name, obj=None, schema=None):
+        """Create a table."""
+        if obj is None and schema is None:
+            raise com.IbisError('Must pass expr or schema')
```

Comment:
test that hits this?

Reply:
why isn't obj required?

Reply:
oh i c you can just pass schema ok

Reply:
This was preexisting, I just moved the code into the base class.